components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/DefaultRequestCoordinator.java

```diff
@@ -545,6 +564,27 @@ protected void unwrapResponse(CommonAuthResponseWrapper responseWrapper, String
         }
     }
 
+    private boolean isPromptRequest(HttpServletRequest request) {
+
+        return StringUtils.isNotBlank(request.getParameter(PROMPT_ID_PARAM))
+                && Boolean.parseBoolean(request.getParameter(PROMPT_RESP_PARAM))
+                && StringUtils.isBlank(request.getParameter(SESSION_LIMIT_HANDLER_PARAM));
```

Comment:
Shall we add a comment explaining why we need to exclude session limit handler as well?
Will be helpful for future references.

Reply:
Fixed via https://github.com/wso2/carbon-identity-framework/pull/6647/commits/3590e96d0bfa7da60f479976c2228fe38c418a89